.travis.yml

```diff
@@ -2,7 +2,8 @@ sudo: false
 language: ruby
 rvm:
   - 2.4.1
-before_install: gem install bundler -v 1.14.6
+  - 2.6.5
```

Comment:
It  would be nice to cover all actual ruby versions :)

- 2.4.7
- 2.5.7
- 2.6.5
- 2.7.0
- ruby-head

Reply:
Done.